ci/jenkins_build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -euo pipefail
+
+# Test configuration environment:
+# KUBE_NODE_TAG:   Docker tag for the Kubernetes node image. This
+#                  determines the Kubernetes version of the KinD cluster.
+#                  Available versions can be found here:
+#                      https://github.com/kubernetes-sigs/kind/releases
+# KUBECTL_VERSION: Version of kubectl to build into test client.
+# CLUSTER_ID:      Cluster identifier string. This will be combined with the
+#                  GIT_REF to generate a unique test client image tag and
+#                  a unique KinD cluster name for this test run.
+# GIT_REF:         Git reference for Conjur OSS helm chart branch being
+#                  tested.
+export KUBE_NODE_TAG=${KUBE_NODE_TAG:-v1.17.5}
+KUBECTL_VERSION=${KUBECTL_VERSION:-1.17.5}
+CLUSTER_ID=${CLUSTER_ID:-1.17.5}
+GIT_REF=${GIT_REF:-master}
+
+echo "==================================="
+echo "Test Configuration:"
+echo "KUBE_NODE_TAG:   $KUBE_NODE_TAG"
+echo "KUBECTL_VERSION: $KUBECTL_VERSION"
+echo "CLUSTER_ID:      $CLUSTER_ID"
+echo "GIT_REF:         $GIT_REF"
+echo "==================================="
+
+KIND_VERSION=0.7.0
+export CLUSTER_NAME="${CLUSTER_ID}-${GIT_REF}"
```

Comment:
It would be good to see the `export`ed variables grouped in a single spot instead of spread around the file